### PR TITLE
Refine the case detail , study detail pages' style

### PR DESCRIPTION
**Pull request by jonkiky**

### CODE REVIEW

File: src/main/frontend/src/pages/caseDetail/caseDetailView.js [hunk 1]
```diff
@@ -65,6 +85,29 @@ const CaseDetail = ({ classes, data }) => {
   const caseDetail = data.case[0];
 
   const notProvided = '';
+
+  const breadCrumb = (       <div className={classes.headerNav}>
```
Comment (AjayDoddapaneni) [inline src/main/frontend/src/pages/caseDetail/caseDetailView.js]: Can you build breadCrumb as a seperate component as this is going to be reused in multiple pages?

Comment (jonkiky) [reply src/main/frontend/src/pages/caseDetail/caseDetailView.js]: But the breadCrumb for each page is different and action is different as well.

Comment (AjayDoddapaneni) [reply src/main/frontend/src/pages/caseDetail/caseDetailView.js]: I understand that,  but you need to build a breadcrumb component such a way that it should display the breadCrumbs depending on the props we are passing

Comment (AjayDoddapaneni) [reply src/main/frontend/src/pages/caseDetail/caseDetailView.js]: And you can use the material UIBreadCrumb if you wish https://material-ui.com/components/breadcrumbs/

**Pull request merged**